pallets/dao/src/lib.rs

```diff
@@ -543,9 +502,12 @@ pub mod pallet {
 						org.description = d;
 					}
 					if let Some(v) = vision {
-						org.vision =v;
```

Comment:
Should we not be checking whether the vision has actually changed before updating these fields? Otherwise the updated_on/by flags will be updated on any change to org - e.g. update to org description.

Reply:
when reading this i was under the impression that None will signify no change. But this is probably wrong as the form will auto populate. I will add this change!

Reply:
Apologies, I missed the Option<> on all the parameters and therefore the implied functionality that values should only be passed in if they've changed. LGTM!